src/types/value/linkToMedia.ts

```diff
@@ -24,4 +24,5 @@ export interface FilledLinkToMediaField {
 	size: string
 	height?: string | null
 	width?: string | null
+	text?: string
```

Comment:
Same comment as @lihbr here: https://github.com/prismicio/prismic-client/pull/348/files#r1716651942

Reply:
Done ✅

Reply:
Actually, I reverted this back after discussing with @xrutayisire & @dani-mp. We intentionally want to keep this as a string and don't intend for it to behave in the same way as a key text field.